src/components/__tests__/__snapshots__/Footer.test.js.snap

```diff
@@ -0,0 +1,54 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Footer Footer snapshot 1`] = `
+<div
+  id="footer"
+>
+  <div
+    className="block-center"
+  >
+    <div
+      className="social-icon-group"
+    >
+      <table>
+        <tr>
+          <td>
+            <a
+              className="social-icon"
+              href="https://github.com/5-gwoap"
+            >
+              <i
+                className="fa fa-github fa-2x"
+              />
+            </a>
+          </td>
+          <td>
+            <a
+              className="social-icon"
+              href="mailto:5gwoap@gmail.com"
+            >
+              <i
+                className="fa fa-envelope fa-2x"
+              />
+            </a>
+          </td>
+        </tr>
+      </table>
+    </div>
+  </div>
+  <div
+    className="block-center"
+  >
+    <div
+      className="footer-text"
```

Comment:
@jwu910 I personally prefer this to be on the same line as the icons. E.G. https://gyazo.com/53de944ab4fd43ea5508351473e35ee4
```
<table><tbody><tr><td><a href="https://github.com/5-gwoap" class="social-icon"><i class="fa fa-github fa-2x"></i></a></td><td><div class="footer-text">Code written under <a href="https://opensource.org/licenses/mit-license.html">MIT license</a></div></td><td><a href="mailto:5gwoap@gmail.com" class="social-icon"><i class="fa fa-envelope fa-2x"></i></a></td></tr></tbody></table>
```
> This just moves the license to the middle. Other options are to move it to either end, but this is just personal pref, no deal breaker